packages/ui-library/src/assets/background-grid.tsx

```diff
@@ -0,0 +1,381 @@
+        fill="url(#paint0_linear_6285_4222)"
+        mask="url(#path-1-inside-1_6285_4222)"
+      />
+      <defs>
+        <linearGradient
+          id="paint0_linear_6285_4222"
+          x1="387.762"
+          y1="0.882812"
+          x2="387.762"
+          y2="199.046"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stopColor="#FCFAFF" stopOpacity="0" />
```

Comment:
should this colors be defined here? what about the theme?

Reply:
Absolutely right